plugin/editorconfig.vim

```diff
@@ -390,11 +433,6 @@ augroup END
 function! s:UseConfigFiles_Python_Builtin() " {{{2
 " Use built-in python to run the python EditorConfig core
 
-    " ignore buffers that do not have a file path associated
```

Comment:
This block is removed because the same check has been already done in `s:UseConfigFiles()` before calling this function.